Cargo.lock

```diff
@@ -3844,10 +3880,20 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "f842b1982eb6c2fe34036a4fbfb06dd185a3f5c8edfaacdf7d1ea10b07de6252"
 dependencies = [
  "lock_api",
- "parking_lot_core",
+ "parking_lot_core 0.6.2",
  "rustc_version",
 ]
 
+[[package]]
+name = "parking_lot"
+version = "0.10.2"
```

Comment:
It seems this update will introduce duplicated versions of `parking_lot` 👀

Reply:
This PR should be edited and a whitelist entry be added. Filed #26262. This will be deduplicated after dependabot upgraded gfx-rs ([wgpu-core git](https://github.com/servo/servo/blob/c78f0d74490d310f6342f7c7af74cde4c5cfbcc8/components/webgpu/Cargo.toml#L21) ([upstream](https://github.com/gfx-rs/wgpu/blob/77d95f5ecdce6a78f1803c4644fd4a245a7d9757/wgpu-core/Cargo.toml#L34) is 0.10), gfx-backend-metal + gfx-backend-dx11 from 0.4 to [0.5](https://crates.io/crates/gfx-backend-metal)), after [surfman](https://github.com/servo/surfman/blob/fde88984c43e8778ae17433eaf4335264c1e758c/surfman/Cargo.toml#L36) upgraded parking_lot to 0.10 and winit to 0.22 and after [webxr](https://github.com/servo/webxr/blob/c0c1e794609c992902efbb54fa64404a69b5ddbe/webxr-api/Cargo.toml#L32) upgraded winit to 0.22.